src/result_collection.jl

```diff
@@ -0,0 +1,92 @@
+using DataFrames
+using BSON
+
+function merge_df(df1, df2)
+    if names(df1) == names(df2)
+        return vcat(df1, df2)
+    else
+        for m ∈ setdiff(names(df1), names(df2))
+            df2[m] = [missing]
```

Comment:
Here why does the `missing` entry need to be in a vector?

Reply:
I'm adding a _column_ to a `DataFrame`.
This is a column with just one single row but it need to be a column-vector nonetheless